enterprise/api/v1/permissions.py

```diff
@@ -30,38 +24,6 @@ def has_permission(self, request, view):
         )
 
 
-class IsAdminUserOrInGroup(permissions.IsAdminUser):
-    """
-    Find out if the requesting user is either staff or belongs to specific django group.
-    It will return a 403 forbidden response with a message if the user is not authorized to access the view.
-    """
-    ALLOWED_API_GROUPS = []  # pylint: disable=invalid-name
-    message = u'User is not allowed to access the view.'
-
-    def has_permission(self, request, view):
-        """
-        Returns True if the requesting user is either 'staff' or belong to specific group, False otherwise.
-        It will also check for group membership against a list of groups in the permissions query parameter.
-        """
-        if waffle.switch_is_active(ENTERPRISE_ROLE_BASED_ACCESS_CONTROL_SWITCH):
-            return True
-
-        return (
-            super(IsAdminUserOrInGroup, self).has_permission(request, view) or
-            request.user.groups.filter(name__in=self.ALLOWED_API_GROUPS).exists() or
-            request.user.groups.filter(name__in=request.query_params.getlist('permissions')).exists()
-        )
-
-
-class HasEnterpriseEnrollmentAPIAccess(IsAdminUserOrInGroup):
-    """
-    Find the requesting user has access to the Enterprise Enrollment API feature set.
-    """
-    def __init__(self):
-        """ Initialize the class with a API_ALLOWED_GROUPS """
-        self.ALLOWED_API_GROUPS = [u'enterprise_enrollment_api_access', ]  # pylint: disable=invalid-name
-
-
 class HasEnterpriseDataAPIAccess(IsInEnterpriseGroup):
```

Comment:
can this class be deleted too?

Reply:
following up - is there a reason we are keeping this class around?

Reply:
This class is used by tests [here](https://github.com/edx/edx-enterprise/blob/master/tests/test_enterprise/api/test_permissions.py#L88-L95)

Reply:
Removed the class and made equivalent changes in tests.